src/WooCommerceShippingPackages.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/**
+ * @file
+ * Contains \Netzstrategen\ShopStandards\WooCommerceShippingPackages.
+ */
+
+namespace Netzstrategen\ShopStandards;
+
+/**
+ * Shipping related settings and actions.
+ */
+class WooCommerceShippingPackages {
+
+  /**
+   * Initialize the needed functions and hooks.
+   */
+  public static function init() {
+    // Show multiple shipping methods separately in order emails.
+    add_filter('woocommerce_get_order_item_totals', __CLASS__ . '::woocommerce_get_order_item_totals', 10, 2);
+  }
+
+  /**
+   * Calculates the shipping costs including tax.
+   *
+   * @param \WC_Order_Item_Shipping $shipping_method
+   *   A shipping item object derived from a WC_Order.
+   *
+   * @return string
+   *   The shipping costs including taxes, currency sign, and HTML line-breaks,
+   *   for output in the HTML order emails.
+   */
+  public static function getShippingCostsIncludingTaxes(\WC_Order_Item_Shipping $shipping_method): string {
+
+    if ($shipping_price) {
```

Comment:
```suggestion
    if ($shipping_price) {
```

I am not sure what are you checking here. In any case, you should check $shipping_method & $order.